File: testresources/java/accuracy/src/domain/indirect/allowedfrom/AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaHeuristic.java
package domain.indirect.allowedfrom;

import domain.indirect.BaseIndirect;
import domain.indirect.intermediate.ServiceTwo;

public class AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaHeuristic {
	ServiceTwo st = new ServiceTwo();
	
	public String initializeProfileInformation(){
		BaseIndirect baseClass = new BaseIndirect();
		Object o = baseClass.getMethodCorrectlyByHeuristic("tsja", st.getServiceOne(), 3);
		return o;
	}
}

File: src/test/resources/csharp/benchmark/Domain/Indirect/AllowedFrom/AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaHeuristic.cs
using Domain.Indirect;
using Domain.Indirect.Intermediate;

namespace Domain.Indirect.AllowedFrom
{

	public class AccessObjectReferenceIndirect_AsReturnValue_MethodDerivedViaHeuristic
	{
		ServiceTwo st = new ServiceTwo();

	public string initializeProfileInformation() {
		BaseIndirect baseClass = new BaseIndirect();
		Object o = baseClass.getMethodCorrectlyByHeuristic("tsja", st.getServiceOne(), 3);
		return o;	
	}

	}
}
